src/crafting_gui.cpp

```diff
@@ -1734,20 +1776,23 @@ static void draw_recipe_tabs( const catacurses::window &w, const std::string &ta
 
     switch( mode ) {
         case NORMAL: {
-            draw_tabs( w, normalized_names, craft_cat_list, tab );
-            int pos_x = 2;
-            for( const std::string &cat : craft_cat_list ) {
-                pos_x += utf8_width( normalized_names[cat] ) + 3;
+            std::map<std::string, std::string> flagged_names = normalized_names;
+            for( const auto &cat : craft_cat_list ) {
                 if( unread[cat] ) {
-                    mvwprintz( w, point( pos_x - 2, 1 ), c_light_green, "⁺" );
+                    auto it = flagged_names.find( cat );
+                    cata_assert( it != flagged_names.end() );
+                    ( *it ).second += "<color_green>*</color>";
```

Comment:
Is there a reason `⁺` is changed to `*` here?

Reply:
...huh.
The new version is a * because I always read the previous version as a *.
If you want a quick PR to change it back to a ⁺, let me know.  I have no particular attachment to the *.

Reply:
I'll change it back to a `⁺` for consistency with the subtabs then.